src/django/contricleaner/lib/handlers/handler.py

```diff
@@ -0,0 +1,14 @@
+from abc import ABC, abstractmethod
+from typing import List
+from contricleaner.lib.dto.list_dto import ListDTO
+
+
+class Handler(ABC):
+
+    @abstractmethod
+    def setNext(self, next):
```

Comment:
1. Please use Pythonic style for function naming: `set_next`.
2. Please add a type hint for the `next` parameter.
3. Please implement this function directly in this class. Though this class will be more like an abstract class than an interface, it is acceptable since child classes will not need to implement the same code repeatedly.